code/e2e-tests/framework-svelte.spec.ts

```diff
@@ -37,4 +38,16 @@ test.describe('Svelte', () => {
     const argsTable = root.locator('.docblock-argstable');
     await expect(argsTable).toContainText('Rounds the button');
   });
+
+  test('Decorators are excluded from generated source code', async ({ page }) => {
+    const sbPage = new SbPage(page);
+
+    await sbPage.navigateToStory('stories/renderers/svelte/slot-decorators', 'docs');
+    const root = sbPage.previewRoot();
+    const showCodeButton = (await root.locator('button', { hasText: 'Show Code' }).all())[0];
+    await showCodeButton.click();
+    const sourceCode = root.locator('pre.prismjs');
+    const expectedSource = '<ButtonJavaScript primary/>';
+    await expect(sourceCode.textContent()).resolves.toContain(expectedSource);
+  });
```

Comment:
I'm hoping to write framework-agnostic versions of this test soon, because I sense that this is an area where we're often seeing regressions.